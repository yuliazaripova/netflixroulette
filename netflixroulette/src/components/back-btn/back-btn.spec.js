import * as React from 'react';

import BackBtn from './back-btn';

describe("BackBtn component", () => {
  it("should render BackBtn component", () => {
    const component = shallow(<BackBtn />);
    expect(component).toMatchSnapshot();
  })
  it("should handle click", () => {
    const handleClick = jest.fn();
    const component = shallow(<BackBtn onClick={handleClick}/>);
    component.simulate('click');
    expect(handleClick.mock.calls.length).toEqual(1);
  })
})