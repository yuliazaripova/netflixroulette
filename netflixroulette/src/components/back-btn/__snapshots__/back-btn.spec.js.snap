// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackBtn component should render BackBtn component 1`] = `
<button
  className="back-btn"
  type="button"
>
  Search
</button>
`;
